### ISSUE TITLE
Build failure

### ISSUE THREAD

**Issue by junh-ki** [eclipse/kuksa.val#68]
Hello :)
I am trying to build the latest version.
I made sure the following things:

1. I've cloned the repository recursively
$ git clone --recursive https://github.com/eclipse/kuksa.val.git

2. I've installed dependencies
$ sudo apt-get update
$ sudo apt-get install libblkid-dev e2fslibs-dev libboost-all-dev libaudit-dev libssl-dev mosquitto libmosquitto-dev

And I've received the following error:

```
junh-ki@junhki-VirtualBox:~/kuksa.val/build$ make
[  5%] Checking the git repository for changes...
[  5%] Built target check_git
[ 68%] Built target kuksa-val-server-object
[ 73%] Built target kuksa-val-server-core
[ 78%] Built target kuksa-val-server-core-static
[ 84%] Building CXX object src/CMakeFiles/kuksa-val-server.dir/main.cpp.o
In file included from /home/junh-ki/kuksa.val/src/main.cpp:39:0:
/home/junh-ki/kuksa.val/src/../buildinfo.h:31:40: error: stray ‘@’ in program
 Signed-off-by: Phong Tran <tranmanphong@gmail.com>

```

Is there anything I am missing here?

**Comment by SebastianSchildt** [comment]
I have the feeling that is related to out cmake magic extracting git build numbers. @phongt  also stumbled upon this. Can you post the `/home/junh-ki/kuksa.val/src/../buildinfo.h` file?

Just a wild guess, maybe it doesn't like several singed-off" in one commit message. To test, you might want to delte buildinfo, and try doing a new commit before building (i.e. just add a space to a file, or add an empty file and git add and commit) Does that work?
@wenwenchenbosch  any ideas?

**Comment by junh-ki** [comment]
/home/junh-ki/kuksa.val/src/../buildinfo.h)


```
// This file was create automatically by CMake.
#pragma once

// Whether or not we retrieved the state of the repo.
#define GIT_RETRIEVED_STATE true

// The SHA1 for the HEAD of the repo.
#define GIT_HEAD_SHA1 "32e850a013df34d70c119553563b69781c6b7649"

// Whether or not there were uncommited changes present.
#define GIT_IS_DIRTY false

// The name of person who committed HEAD.
#define GIT_AUTHOR_NAME "Wenwen Chen"

// The email addess of the person who committed HEAD.
#define GIT_AUTHOR_EMAIL "68554424+wenwenchenbosch@users.noreply.github.com"

// When HEAD was committed.
#define GIT_COMMIT_DATE_ISO8601 "2020-11-06T16:28:54+01:00"

// The subject line for the HEAD commit.
#define GIT_COMMIT_SUBJECT "64 unittest ci (#65)"

// The notes attached to the HEAD commit.
// #define GIT_COMMIT_BODY "* README: Add installation libmosquitto-dev

Without installation of Mosquitto dependency will build error.

Signed-off-by: Phong Tran <tranmanphong@gmail.com>

* Fix unit test compile failure

Signed-off-by: Phong Tran <tranmanphong@gmail.com>

* add test stage and enable run ctest

Signed-off-by: Chen Wenwen (CR/AME5) <Wenwen.Chen@de.bosch.com>

* fix jenkinsfile

Signed-off-by: Chen Wenwen (CR/AME5) <Wenwen.Chen@de.bosch.com>

* revert changes in test

Signed-off-by: Chen Wenwen (CR/AME5) <Wenwen.Chen@de.bosch.com>

* do not run ctest

Signed-off-by: Chen Wenwen (CR/AME5) <Wenwen.Chen@de.bosch.com>

* rename and build test in docker

Signed-off-by: Chen Wenwen (CR/AME5) <Wenwen.Chen@de.bosch.com>

* disable running test

Signed-off-by: Chen Wenwen (CR/AME5) <Wenwen.Chen@de.bosch.com>

Co-authored-by: Phong Tran <tranmanphong@gmail.com>"

```

**Comment by SebastianSchildt** [comment]
Delete `buildinfo.h`

Then edit `buildinfo.h.in` and REMOVE the last two lines 

```
// The notes attached to the HEAD commit.
// #define GIT_COMMIT_BODY "@GIT_COMMIT_BODY@"
```

Then cmake & make again.

Our cmake tooling break when the last commit has more than one line. Since we do not use commit_body (message) in our code, it is ok, to dlete it in the .in file

If you have time, please make a pullrequest with the two lines removed from  .in file 😄

**Comment by SebastianSchildt** [comment]
Closed by #70 
@junh-ki  can you close this issue? (I can't 😁 )

**Comment by junh-ki** [comment]
> Closed by #70
> @junh-ki can you close this issue? (I can't 😁 )

I was waiting for the build result and it just built it successfully so I am closing :D
